common/workunit/workunit.cpp

```diff
@@ -6934,7 +6934,7 @@ wuTokenStates verifyWorkunitDAToken(const char * distributedAccessToken)
     {
         if (!streq(cw->queryUser(), tokUser.str()))
         {
-            ERRLOG("verifyWorkunitDAToken : token user does not match workunit");
+            OERRLOG("verifyWorkunitDAToken : token user does not match workunit");
```

Comment:
Not sure it is operator, possibly audit though - suggests something fishy going on.